build.yaml

```diff
@@ -102,7 +102,24 @@ release:
                   find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
                   rm -rf build
                   PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
-                  sudo ${PACKAGE_UTILITY_REMOVE} ${PHP_TOKENS[0]}-common
+                  # Installing packaging and testing connection
+                  sudo ${PACKAGE_INSTALL} packages/${PHP_TOKENS[0]}*.${PACKAGE_TYPE}
+                  php <<EOF
+                  <?php
+                  \$cluster = Cassandra::cluster()->build();
```

Comment:
For the sake of simplicity and  to reduce escaping you could remove the `$cluster` temporary variable.